### ISSUE TITLE
Providing a sitetree with ones website ?

### ISSUE THREAD

**Issue by Carelvd** [idlesign/django-sitetree#259]
Thank you for sitetrees it is a really nice package. 

I have a minor issue though and I'm not sure if I've messed up something or if this is a bug. I have done the following :

    python -m django startproject website
    move website PROJECT
    cd PROJECT python manage.py startapp APPLICATION
   
To which I add a `sitetrees.py` file both under the application and the website as follows :

    PROJECT\
      website\
        ...
        sitetrees.py
      APPLICATION\
        ...
        sitetrees.py

Within `website\sitetrees.py` I have added the following :

    from sitetree.utils import tree, item
    print("Website:SiteTrees")

    navigation = (
        item('Home',  'home' , hint = "Landing Page", ),
        item('About', 'about', hint = "General Information", access_guest = True),
       )

    sitetrees = (tree('website_navigation', items=navigation),)

Within `APPLICATION\sitetrees.py` I have added the following :

    from sitetree.utils import tree, item
    print("APPLICATION:SiteTrees")

    navigation = (
        item('Home',  'home' , hint = "Landing Page", ),
        item('About', 'about', hint = "General Information", access_guest = True),
        )

    sitetrees = (tree('website_navigation', items=navigation),)

I then synchronize the trees as per the documentation 

    (venv) E:\Django\PROJECT>python manage.py sitetree_resync_apps
    APPLICATION:SiteTrees
    Sitetrees found in `APPLICATION` app ...
      Processing `APPLICATION_navigation` tree ...
        Adding `Home` tree item ...
        Adding `About` tree item

The resulting tree however is empty

![sitetree](https://user-images.githubusercontent.com/322018/51588559-437f2800-1eed-11e9-80d8-d639f92f7091.png)
 
Usually adding something under `PROJECT/website` is asking for trouble, since then you have to add `website` as an application under `INSTALLED_APPS` which, as I understand it is a bit of a no no. So I'm not expecting `PROJECT/website/sitetrees.py` to synchronize with the tree but I was rather hoping that `PROJECT/APPLICATION/sitetrees.py` would work and it appears not to do so. Have I messed up here or is there a bug I've discovered ? 

I'm using Python version 3.6.3 and Django 2.1..5

**Comment by Carelvd** [comment]
I'm fine with the project level not being checked (I rather expected that) but if I wipe the DB, resync sitetrees and start the server again and I still get an empty tree within the database as shown in the original image. Is this the expected behaviour ? What I'm not certain of is that the `APPLICATION_navigation` tree is empty ?

**Comment by Carelvd** [comment]
Do you mean I should interrogate the model directly through the shell ? (I'm not sure what you mean by `sitetree_items`)

**Comment by idlesign** [comment]
Yes you can use Django shell or pip into database table named ``sitetree_items`` directly using DB shell.

**Comment by Carelvd** [comment]
So interrogating the model from the shell gives me the two items I specified in `APPLICATION/sitetrees.py`, I have now removed `website/sitetree.py`, wiped the database, migrated, and re-synchronized sitetrees. 

    >>> from sitetree.models import TreeItem
    >>> TreeItem.objects.all()
    <QuerySet [<TreeItem: Home>, <TreeItem: About>]>
    >>> for i in TreeItem.objects.all() :
    ...   print(i)
    ...
    Home
    About

I can see the tree `navigation` in the admin interface. Adding a new item to the tree in the admin interface shows that the items I've specified in `sitetrees.py` may be used as potential parents, but the items themselves are not visible in the tree directly (That is they are not shown in the foreign key/reverse look up table ?).

**Comment by idlesign** [comment]
> It seems the tree itself is recreated anew every time I re-sync.

Yes. That's how it works.

> Adding a new item to the tree in the admin interface shows that the items I've specified in sitetrees.py may be used as potential parents, but the items themselves are not visible in the tree directly.

Sounds like a permission-related problem. I'll try to investigate the issue this week.

**Comment by idlesign** [comment]
So at last I have had a chance to check this issue using demo project — https://github.com/idlesign/django-sitetree/tree/master/demo — and that's what I see after syncing:

![shot](https://user-images.githubusercontent.com/145644/51980600-15887d80-24c3-11e9-9383-cdf61bbbd6f4.png)

Please try the demo with your code placed in https://github.com/idlesign/django-sitetree/blob/master/demo/demo/sitetrees.py to verify whther the issue persists.
